docs/en/observability/quickstarts/collect-data-with-aws-firehose.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[collect-data-with-aws-firehose]]
+= Quickstart: Collect data with AWS Firehose
+
+preview::[]
+
+In this quickstart guide, you'll learn how to send logs and metrics to Elastic using AWS Firehose.
+
+The AWS Firehose receiver is based on a CloudFormation template that can collect all available CloudWatch logs and metrics for your AWS account.
+
+This approach requires minimal configuration as the CloudFormation template creates a Firehose stream, enables CloudWatch metrics collection, and sets up account level subscription filter for CloudWatch log groups to send logs to Elastic via Firehose.
+
+NOTE: Optionally, you can download the CloudFormation template to review and modify the default settings. For more information, check the 
+link:https://www.elastic.co/docs/current/integrations/aws[AWS integration] documentation.
```

Comment:
I don't think this is correct. AWS integration and the template aren't really connected in this context, I'd suggest to separate them and have AWS Integration reference as a separate paragraph that states that some AWS services need additional manual configuration in order to properly ingest logs/metrics, it's an important point we need to emphasize.

Reply:
Agree, we should have a separate paragraph for all optional parameters and how to use them.

Reply:
I reworded the text based on your comments. Please review and see if it is any better.